server/middleware/setUpStaticResources.ts

```diff
@@ -37,6 +37,30 @@ export default function setUpStaticResources(): Router {
   ;['/node_modules/jquery-ui-dist/jquery-ui.min.css'].forEach(dir => {
     router.use('/assets/stylesheets/jquery-ui.min.css', express.static(path.join(process.cwd(), dir), cacheControl))
   })
+  ;['/node_modules/@ministryofjustice/hmpps-digital-prison-reporting-frontend/dpr/assets'].forEach(dir => {
+    router.use('/assets/dpr', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/chart.js/dist/chart.umd.js'].forEach(dir => {
+    router.use('/assets/ext/chart.js', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/chart.js/dist/chart.umd.js.map'].forEach(dir => {
+    router.use('/assets/ext/chart.umd.js.map', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/chartjs-plugin-datalabels/dist/chartjs-plugin-datalabels.min.js'].forEach(dir => {
+    router.use('/assets/ext/chartjs-datalabels.js', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/dayjs/dayjs.min.js'].forEach(dir => {
+    router.use('/assets/ext/day.js', express.static(path.join(process.cwd(), dir), cacheControl))
+  })
+  ;['/node_modules/dayjs/plugin/customParseFormat.js'].forEach(dir => {
+    router.use(
+      '/assets/ext/dayjs/plugin/customParseFormat.js',
+      express.static(path.join(process.cwd(), dir), cacheControl),
+    )
+  })
+  ;['/node_modules/govuk-frontend/dist/govuk/govuk-frontend.min.js'].forEach(dir => {
```

Comment:
**issue:** we already have an entry for govuk-frontend above.

**suggestion:** We don't need to iterate over arrays of one item. 
It'd be better to iterate over all items or if that's not possible then just inline each call

Reply:
Cool, have updated to no longer iterate where not needed and removed duplication of entry for govuk-frontend - sorry thats me thinking the second was different, introduced via the DPR guidance.